### ISSUE TITLE
xlrd.xldate.XLDateAmbiguous err

### ISSUE THREAD

**Issue by PPTA22** [snoopyjc/xls2xlsx#5]
* xls2xlsx version: 0.1.3
* Python version: 3.7.9
* Operating System: Win 10 Pro, 1909

### Description

`xlrd.xldate.XLDateAmbiguous` raised while converting to xlsx

I have tried to use the `Save as` to output as another xls file in the Excel 2010, but the same error still occur

### What I Did

```
>>> from xls2xlsx import XLS2XLSX
>>> os.listdir()
['TSLINES HKG.xls']
>>> x2x = XLS2XLSX(os.listdir()[0])
>>> wb = x2x.to_xlsx()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\#####\Anaconda3\envs\ocean2\lib\site-packages\xls2xlsx\xls2xlsx.py", line 214, in to_xlsx
    value = self.xls_date_to_xlsx(value)
  File "C:\Users\#####\Anaconda3\envs\ocean2\lib\site-packages\xls2xlsx\xls2xlsx.py", line 71, in xls_date_to_xlsx
    date_tuple = xlrd.xldate_as_tuple(value, self.date_mode)
  File "C:\Users\#####\Anaconda3\envs\ocean2\lib\site-packages\xlrd\xldate.py", line 116, in xldate_as_tuple
    raise XLDateAmbiguous(xldate)
xlrd.xldate.XLDateAmbiguous: 7.0
```

**Comment by PPTA22** [comment]
[TSLINES HKG.zip](https://github.com/snoopyjc/xls2xlsx/files/5479319/TSLINES.HKG.zip)

**Comment by snoopyjc** [comment]
Excel has a bug it inherited from Lotus 1-2-3 where it thinks there is a Feb 29, 1900.  This is called the Excel Leap Year bug.  The xls reader I use (xlrd) raises this XLDateAmbiguous error when attempting to read dates between Jan 1, 1900 and Mar 1, 1900.  I'll look to fix this eventually, as I plan to make a clone of xlrd (xlrd is a dead project right now).

**Comment by snoopyjc** [comment]
On second thought, I should be able to come up with a quick patch for you for this tomorrow, as I have this issue solved in my new ssf (SpreadSheet Formatter) program.  Root cause is this code in xlrd:

    if xldays < 61 and datemode == 0:
        raise XLDateAmbiguous(xldate)

**Comment by PPTA22** [comment]
That's great, thank you so much

**Comment by snoopyjc** [comment]
On second thought, I should be able to come up with a quick patch for you for this tomorrow, as I have this issue solved in my new ssf (SpreadSheet Formatter) program.  Root cause is this code in xlrd:

    if xldays < 61 and datemode == 0:
        raise XLDateAmbiguous(xldate)

**Comment by snoopyjc** [comment]
As an avoidance, you can clear out the very old dates in hidden cells V46:V53.

**Comment by PPTA22** [comment]
That can be one of the solution. But I am not sure will there be any formula based on the V46:V53 value. I have to look into it later. 
As these excel are generated by the shipment company, not sure will there any formula behind it.
Thanks

**Comment by snoopyjc** [comment]
Just clear out the Formats for them then - the value "7.0" is formatted as a date, which causes the issue.  I should have a fix for you later this evening, though.

**Comment by snoopyjc** [comment]
Fixed in v0.1.5
